AmplifyTools/AmplifyCLIiOS/Sources/AmplifyCLIiOS/Core/Commands/CommandImportConfig.swift

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+func amplifyFolderExist(
```

Comment:
nit: this and `addConfigFilesToXcodeProject...` don't feel like top level functions. Are they so generally applicable to this module that it makes sense for them to be in the main module namespace as opposed to segregated into their own namespace? E.g. a common pattern for defining namespaces for grouping is to use an enum with no cases.